### ISSUE TITLE
404 and 503 errors across random pages

### ISSUE THREAD

**Issue by skorozsi** [kimai/kimai#5159]
### Describe the issue

I installed Kimai on a subdomain through my hosting provider using cPanel and Softaculous. The site runs on a LiteSpeed Web Server. Initially, Kimai worked well, but after a while, certain pages began showing a 503 error and stopped loading.

I followed the documentation on installation and permissions and set the necessary permissions as advised. Logging only started functioning after I enabled log rotation yesterday, so now I have logs, but they only contain debug and info entries.

Until today, I could temporarily fix non-loading pages by clearing the cache via SSH connection. However, this solution was inconsistent, as the 503 or 404 errors would appear on random pages each time. While inconvenient, I could still work around it.

Today, however, the entire site stopped working after I edited my lastly recorded  manual time entry's  length on the timesheet page. Now, clearing the cache doesn’t help, and every page request returns a 404 error.

I have no any idea what to check yet. I checked and read all 404 bug reports here, etc. 
Could you please suggest any other steps to check/read another docs etc?

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.24.0

### How do you run Kimai?

Shared-Hosting

### Which PHP version are you using?

8.1

### Logfile

[2024-11-14T11:14:54.917178+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"secured_area","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} {"channel":"security"}
[2024-11-14T11:14:54.917200+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"secured_area","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} {"channel":"security"}
[2024-11-14T11:14:54.917219+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"secured_area","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator"} {"channel":"security"}
[2024-11-14T11:14:54.917262+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"secured_area","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator"} {"channel":"security"}
[2024-11-14T11:27:41.917021+00:00] deprecation.INFO: User Deprecated: Since doctrine/doctrine-bundle 2.13: Enabling the controller resolver automapping feature has been deprecated. Symfony Mapped Route Parameters should be used as replacement. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since doctrine/doctrine-bundle 2.13: Enabling the controller resolver automapping feature has been deprecated. Symfony Mapped Route Parameters should be used as replacement. at /home/username/domain/vendor/doctrine/doctrine-bundle/src/DependencyInjection/DoctrineExtension.php:533)"} {"channel":"deprecation"}

### Screenshots

![image](https://github.com/user-attachments/assets/0df92cf5-3eb1-45b8-9458-d33127bd8d30)

**Comment by kevinpapst** [comment]
Sorry to hear about your bad experience.
Please share the name of the Hoster, so we can add it to the docs as nono-functioninig.

I will close here as this is not a Kimai issue, but the cause is a incompatible (or shitty) environment.

You can open always a discussion and see if you find someone with experience with this hoster.

**Comment by skorozsi** [comment]
ok, thanks for your time!
This is my provider: [sybell.hu](https://www.sybell.hu)
I am going to investigate it further and will share info if I manage to solve this somehow.

**Comment by kevinpapst** [comment]
If you have at least some logs that show an error, we can talk. 
And I don't understand what log rotating has to do with Kimai.
Kimai stores logs at kimai/var/log/

Also there are some 3rd party PHP Plugins, which tend to screw up the environment. 
But a 404 ist super strange and not explainable by that.

**Comment by skorozsi** [comment]
Hi Kevin, 
Thanks for your answer!

> If you have at least some logs that show an error, we can talk. 
> 
Kimai did not write any log in kimai/var/log until I switched on log rotation in `local.yaml` despite I followed the documentation how to set up permissions. The log file was not even created. I created kimai/var/log/prod.log manually, but it didn't matter for Kimai, there was no any line written in it. 

> And I don't understand what log rotating has to do with Kimai. Kimai stores logs at kimai/var/log/
> 
Because I had no any idea left what to do to get any log, I tried to set up log rotation because it creates separate name (not prod.log). This morning I realized that switching on log rotation somehow enabled the logging which was not working until that change. 
I set up log rotation the way it was described in your documentation, then log files started to appear automagically in the log folder:
`monolog:
    handlers:
        nested:
            type:  rotating_file
            path:  '%kernel.logs_dir%/%kernel.environment%.foooooo.log'
            max_files: 10`
https://www.kimai.org/documentation/logging.html
I don't think it is a permission issue regarding logging. 

> If you have at least some logs that show an error, we can talk. 
> 
I have two log files there, one for tomorrow, one for today, but there is no any error in them unfortunately. 
Since the start of 404 errors these logs didn't even contain any new entries. 

I think I will try to host Kimai on my windows PC somehow. I first tried using Docker version, but as mentioned in the documentation the DB is cleaned when the container is stopped, so I need another approach. 
UPDATE: I just realized that there is a TEST version of docker install and an other. I tried the test version... So I will try the Docker Compose now!
